tests/rotctl_parse.c

```diff
@@ -1472,7 +1472,7 @@ int rotctl_parse(ROT *my_rot, FILE *fin, FILE *fout, const char *argv[],
     pthread_mutex_unlock(&rot_mutex);
 #endif
 
-    if (retcode == RIG_EIO) { return retcode; }
```

Comment:
The new check is ok, but the return is a different bug (it should return `2` meaning Hamlib error, so I'm going to open a new PR).